config/nimbus.yaml

```diff
@@ -138,8 +138,8 @@ features:
         }
       - channel: staging
         value: {
-          "enabled": false,
-          "variant": default,
+          "enabled": true,
+          "variant": redesign,
```

Comment:
Since we rolled out the landing page redesign on production, let’s enable it on stage.